src/Core/PostgreSQL/PoolWithFailover.cpp

```diff
@@ -94,7 +103,9 @@ ConnectionHolderPtr PoolWithFailover::get()
                 catch (const pqxx::broken_connection & pqxx_error)
                 {
                     LOG_ERROR(log, "Connection error: {}", pqxx_error.what());
-                    error_message << "Try " << try_idx + 1 << ". Connection to `" << replica.connection_info.host_port << "` failed: " << pqxx_error.what() << "\n";
+                    error_message << "Try " << try_idx + 1 << ". "
```

Comment:
I understand it changes the existing message, but overall wouldn't it better first write message body and then parameters, so it'd be easier to search in traces later. For example:
```
... str::format("Connection to PostgreSQL failed: try={}, host={}, error={}", ...)
```